numba/tests/test_pycc.py

```diff
@@ -353,13 +353,52 @@ def run_python(args):
             """
         run_python(["-c", code])
 
+    def check_setup_nested_py(self, setup_py_file):
+        # Compute PYTHONPATH to ensure the child processes see this Numba
+        import numba
+        numba_path = os.path.abspath(os.path.dirname(
+                                     os.path.dirname(numba.__file__)))
+        env = dict(os.environ)
+        if env.get('PYTHONPATH', ''):
```

Comment:
I am not sure I understand this logic here. Under what circumstances would the `PYTHONPATH` be set during testing?

Reply:
Please compare with `def check_setup_py` above.